google/ggrc-core
# GGRC-6231 Add functionality for collecting cloud tasks

# Note

This PR require enabling AppEngine cloud task API in cloud console

# Issue description

In some cases we need to have a list of tasks existing in cloud task queue.

# Steps to test the changes

- Allow Cloud Task API for app engine instance.
- Deploy new version with "APPENGINE_LOCATION" in settings (by default us-central1)
- Run some background tasks (for example reindex)
- Check task names collected by `get_app_engine_tasks`, compare it with content of queue in cloud console.

# Solution description

Cloud Task API is used to grab a list of tasks existing in taskqueue currently. For api authentication default service account credentials are applied.

# Sanity checklist


# PR Review checklist
